requirements/base.in

```diff
@@ -3,4 +3,4 @@
 
 django-statici18n
 bleach[css]
-xblock-utils
+XBlock[django]>=1.8.1
```

Comment:
If we have the `xblockutils` fallback, we can remove the constraint.

```suggestion
XBlock[django]
```

Reply:
I have a question here @Agrendalath 
what if someone is installing this xblock directly on the xblock-sdk and this repo is using a lesser version of xblock from v1.8.1. will it not break the xblock? (considering we have already removed xblock-utils package from this xblock)

Reply:
@farhan, I see two possible cases:
1. Installing `xblock-sdk` from PyPI.
    In this scenario, dependencies are specified in`setup.py`, which [pulls them from `requirements/base.in`](https://github.com/openedx/xblock-sdk/blob/796c512/setup.py#L119) that contains [an unpinned version of `XBlock[django]`](https://github.com/openedx/xblock-sdk/blob/796c512/requirements/base.in#L14). Therefore, `pip` pulls the most recent version of `XBlock[django]`, which (at the moment) is `v1.8.1`.
1. Installing `xblock-sdk` dependencies from its requirements (e.g., after cloning the `xblock-sdk` repo for development).
    In this scenario, the developer uses [`make requirements` or `make install`](https://github.com/openedx/xblock-sdk/blob/796c512e5121639c4a15c3dc46659ff418630479/Makefile#L77-L81) _(side note: these recipes should use `pip-sync` to ensure that the removed packages are uninstalled from a virtualenv)_. In this scenario, [we already use the `xblock[django]==1.8.1`](https://github.com/openedx/xblock-sdk/blob/796c512/requirements/dev.txt#L378-L383).

Of course, the second scenario can install an older version if somebody does not use an up-to-date branch of `xblock-sdk`. However, it can be resolved with `git pull`, so it shouldn't be a big deal.